src/skmultiflow/rules/numeric_attribute_class_observer.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+from sortedcontainers.sortedlist import SortedList
+
+from skmultiflow.rules.attribute_expand_suggestion import AttributeExpandSuggestion
+from skmultiflow.trees.attribute_class_observer import AttributeClassObserver
+from skmultiflow.trees.gaussian_estimator import GaussianEstimator
+
+
+class GaussianNumericAttributeClassObserver(AttributeClassObserver):
```

Comment:
What is the difference between `NumericAttributeClassObserverGaussian` defined in the `trees` module and this one?

Reply:
Different functionality